2-collectors/scala-stream-collector/examples/config.hocon.sample

```diff
@@ -114,7 +114,7 @@ collector {
 
     # Enable the chosen sink by uncommenting the appropriate configuration
     sink {
-      # Choose between kinesis, googlepubsub, kafka, nsq, or stdout.
+      # Choose between kinesis, googlepubsub, kafka, nsq, stdout or eventhub.
```

Comment:
:art: could you put eventhub before stdout so that they're all sorted by order of importance :+1: